core/src/main/resources/defaults/clustertemplates/aws/aws-opdb-702.json

```diff
@@ -2,6 +2,7 @@
   "name": "CDP 1.2 - Operational Database template",
   "description": "",
   "type": "OTHER",
+  "release": "RELEASED",
```

Comment:
featureState would be more self-explanatory in my humble opinion if you ask me.

Reply:
cool I will do that